root/static/scripts/common/entity.js

```diff
@@ -3,31 +3,28 @@
 // Licensed under the GPL version 2, or (at your option) any later version:
 // http://www.gnu.org/licenses/gpl-2.0.txt
 
-const ko = require('knockout');
-const _ = require('lodash');
-const React = require('react');
-const ReactDOMServer = require('react-dom/server');
-
-const ArtistCreditLink = require('./components/ArtistCreditLink');
-const EditorLink = require('./components/EditorLink');
-const EntityLink = require('./components/EntityLink');
-const {
-    ENTITY_NAMES,
```

Comment:
ENTITY_NAMES seems to be still needed, see https://github.com/metabrainz/musicbrainz-server/pull/919/files#diff-37e659b91af89773d96c6b5faa90d18cR71

Reply:
Yeah, my mistake, this was probably added to the file after I had written the code changing this and I missed it. Fixed!